bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java

```diff
@@ -87,18 +111,33 @@ public KeyValueStorageRocksDB(String basePath, String subPath, DbConfigType dbCo
         this.emptyBatch = new WriteBatch();
 
         String dbFilePath = "";
+        if (dbConfigType == DbConfigType.EntryLocation) {
+            dbFilePath = conf.getEntryLocationRocksdbConf();
+        } else if (dbConfigType == DbConfigType.LedgerMetadata) {
+            dbFilePath = conf.getLedgerMetadataRocksdbConf();
+        } else {
+            dbFilePath = conf.getDefaultRocksDBConf();
+        }
```

Comment:
I would add a log line
`log.info("Searching for a RockDB configuration file in {}", dbFilePath)`